jobs/involuntary-dissolutions/config.py

```diff
@@ -79,8 +79,8 @@ class _Config(object):  # pylint: disable=too-few-public-methods
 
     SUMMARY_STORAGE_PATH = os.getenv('SUMMARY_STORAGE_PATH', '')
     SECOND_NOTICE_DELAY = os.getenv('SECOND_NOTICE_DELAY', None)
-    STAGE_2_DELAY = os.getenv('STAGE_2_DELAY', None)
-    STAGE_3_DELAY = os.getenv('STAGE_3_DELAY', None)
+    STAGE_2_DELAY = os.getenv('STAGE_2_DELAY', 0)
+    STAGE_3_DELAY = os.getenv('STAGE_3_DELAY', 0)
```

Comment:
Changing these defaults to `0` makes the logic cleaner on lines 115-116 in `involuntary_dissolutions.py`, however if we want to keep it as `None` I can just add checks in that file before creating the `timedelta`. Note that for some reason `app.config.get('STAGE_2_DELAY', 0)` will return `None` if the fallback here is `None`

Reply:
I prefer keeping it as `None` so in code of stage 2, it will skip the run if such config value doesn't exist. This makes more sense to me. 🤔

Reply:
Sounds good, updated!